### ISSUE TITLE
Since updated to 1.6.0 Homebridge is not starting up correctly.

### ISSUE THREAD

**Issue by cebor** [oznu/homebridge-syno-spk#108]
Today i updated Homebridge to 1.6.0 and now its not starting up correctly. The wheel at the top of the default dashboard keeps spinning: 
![image](https://user-images.githubusercontent.com/1576120/204148859-1f5f2ab3-a5f2-4912-8a6a-c6a0403a46cb.png)

My only Accessory (Nuki Lock 2.0) is not showing up in the Accessory List and marked as not responding in Homekit.

Even in debug mode logs seem to be fine.

```log
[11/27/2022, 5:47:54 PM] Homebridge v1.6.0 (HAP v0.11.0) (Homebridge 6B31) is running on port 51844.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
[11/27/2022, 5:47:54 PM] [Nuki] Launched child bridge with PID 14616
[11/27/2022, 5:47:54 PM] The plugin "homebridge-nb" requires Node.js version of ^18.12.1 which does not satisfy the current Node.js version of v16.16.0. You may need to upgrade your installation of Node.js - see https://homebridge.io/w/JTKEF
[11/27/2022, 5:47:54 PM] Registering platform 'homebridge-nb.NB'
[11/27/2022, 5:47:54 PM] [Nuki] Loaded homebridge-nb v1.2.13 child bridge successfully
[11/27/2022, 5:47:54 PM] Loaded 2 cached accessories from cachedAccessories.0E5EB543D43F.
[11/27/2022, 5:47:54 PM] [Nuki] homebridge-nb v1.2.13, node v16.16.0, homebridge v1.6.0, homebridge-lib v6.0.2
[11/27/2022, 5:47:54 PM] [Nuki] warning: recommended version: node v18.12.1 LTS
[11/27/2022, 5:47:54 PM] [Nuki] Nuki_Bridge_41049D55: Nuki Bridge v2.14.0 41049D55 at 10.0.30.22:8080
[11/27/2022, 5:47:54 PM] [Nuki] Haustür: Nuki Smart Lock 2.0 v2.12.4 2AD96924
[11/27/2022, 5:47:54 PM] Homebridge v1.6.0 (HAP v0.11.0) (Nuki) is running on port 37817.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
[11/27/2022, 5:47:54 PM] [Nuki] hardware: Synology Synology NAS RS1221+
[11/27/2022, 5:47:54 PM] [Nuki] os: DSM 7.1.1-42962 Update 2
[11/27/2022, 5:47:54 PM] [Nuki] restored 2 accessories from cache
[11/27/2022, 5:47:55 PM] [Nuki] listening on http://0.0.0.0:41104/notify
[11/27/2022, 5:47:55 PM] [Nuki] Nuki_Bridge_41049D55: subscribe to event notifications
[11/27/2022, 5:47:56 PM] [Nuki] Nuki_Bridge_41049D55: set Last Updated from "Sun Nov 27 2022 17:47:16" to "Sun Nov 27 2022 17:47:50"
```

**Comment by cebor** [comment]
Is this a problem because synology has no node v18? Is there a way go back to 1.5.1?

**Comment by airart320** [comment]
Just change the mDNS advertiser to Ciao in Homebridge settings.

**Comment by cebor** [comment]
> Just change the mDNS advertiser to Ciao in Homebridge settings.

Yes thanks that works.

Is there any technical explanation why it worked before with Avahi, think this was set by default, never changed anything.

**Comment by cebor** [comment]
With the docker image its still working with avahi. Is this an common issue with 1.6.0 or maybe an issue with my synology?

I installed Home Assistant before.

**Comment by cuttalfish** [comment]
Wow so glad I found this. I'm running Homebridge on my Synology for the first time and couldn't figure this out. Thank you!

**Comment by aaronk1** [comment]
> Just change the mDNS advertiser to Ciao in Homebridge settings.

Thanks @airart320 ! Synology DSM 7 also and this fix worked on 1.6.0
